app/editor/src/components/form/datepicker/SelectDate.tsx

```diff
@@ -33,19 +38,35 @@ export const SelectDate: React.FC<IDatePickerProps> = ({
   tooltip,
   children,
   className,
+  selectedDate,
   ...rest
 }) => {
+  const [startDate, setStartDate] = useState(selectedDate ? new Date(selectedDate) : new Date());
+
   return (
     <styled.SelectDate className="frm-in">
       {label && <label htmlFor={id ?? `txt-${name}`}>{label}</label>}
-      <DatePicker
-        name={name}
-        id={id}
-        className={`dpk ${className ?? ''}`}
-        data-for="main-tooltip"
-        data-tip={tooltip}
-        {...rest}
-      />
+      {selectedDate ? (
```

Comment:
What I don't understand is why this statement needs to be here.  Can you just have one `<DatePicker.../>` below?

Reply:
Going to take more of a look at this, I was having issues with providing the `rest` props when providing an `onChange` and some of the conditional rendering. I know I can fix it though, just trying to get the PR in for testing purposes. Was making the `DatePicker` behave in a different way on the list view. Will clean it up though!